evap/staff/templates/staff_user_index.html

```diff
@@ -64,8 +64,7 @@
                                 ></button>
                             {% else %}
                                 <button type="button" disabled class="btn btn-sm btn-danger" data-toggle="tooltip" data-placement="left"
-                                    title="{% blocktrans %}This user contributes to an evaluation (or is a delegate or CC user of
-                                    such a person), participates in an evaluation whose participations haven't been archived yet or has special
+                                    title="{% blocktrans %}This user contributes to an evaluation, participates in an evaluation whose participations haven't been archived yet or has special
```

Comment:
please have a look at the line breaks, they are a bit mixed up here. if easily possible, lines should not be longer than 120 characters.